src/applications/claims-status/components/Breadcrumbs.jsx

```diff
@@ -1,29 +1,38 @@
 import React from 'react';
+import { Link } from 'react-router';
+import Breadcrumbs from '@department-of-veterans-affairs/formation/Breadcrumbs';
+
+class ClaimsBreadcrumbs extends React.Component {
+  renderBreadcrumbs = childNodes => {
+    const crumbs = [
+      <a href="/" key="home">
+        Home
+      </a>,
+      <a href="/disability-benefits/" key="disability-benefits">
```

Comment:
The new link here is actually `/disability/`, but I'm not sure it should be changed b/c it'll break on Vets.gov, at least for the next few days. Thoughts?

Reply:
There's a redirect in place that will make this work correctly in both places for now if you leave it set to the vets.gov path. The other option is to use the brand consolidation flag and switch between them, but we haven't been doing that for links generally.

Reply:
@jbalboni I will follow suit and let the re-direct handle it.